pom.xml

```diff
@@ -18,20 +18,36 @@
     <maven.compiler.target>${version.java}</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <spring-boot.version>2.7.6</spring-boot.version>
-    <gwt.version>2.10.0</gwt.version>
+    <spring-boot.version>3.0.0</spring-boot.version>
+    <gwt-user.version>2.10.0</gwt-user.version>
+    <gwt-servlet.version>2.11.0-jakarta-SNAPSHOT</gwt-servlet.version>
```

Comment:
As this is a public repository, will everyone be able to access this dependency?

Reply:
Yes as the repo where the dependency lies is public